build/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -58,6 +62,7 @@ export default () => {
         variations: {
           name: behaviorVariantName,
           text: description,
```

Comment:
lets rename `text` => `description`, as general `text` is very hard to reason about while reading the following lines: https://github.com/stardust-ui/react/pull/455/files#diff-105d2d2a4ca631ebeb4f7bdafcc0542bR74